src/loaders/types.d.ts

```diff
@@ -5,7 +5,11 @@ import {
 } from 'jwplayer/types';
 
 export type DesktopArticleVideoLoaderProps = DesktopArticleVideoPlayerProps | undefined;
-export type MobileArticleVideoLoaderProps = MobileArticleVideoPlayerProps | undefined;
+export type MobileArticleVideoLoaderProps =
+	| (MobileArticleVideoPlayerProps & {
+			scrollTopPosition?: string;
+	  })
+	| undefined;
```

Comment:
It's a minor issue, but could we improve readability by removing the extra pipe character and adjusting the indentation?

Reply:
I'm pretty sure it was generated by prettier - so I would not touch it.

Reply:
I believe you're pretty right. Thanks for pointing that out!